app/javascript/components/atoms/TableWrapper.js

```diff
@@ -2,8 +2,8 @@ import React from 'react'
 
 const TableWrapper = ({ children }) => (
   <div className='form-grid row expand'>
-    <div className='small-12 column'>
-      <div className='scrollable-table-container-under-xlarge'>
+    <div className='small-12 medium-12 large-12 column'>
```

Comment:
nit: isn't `small-12` the same as `small 12 medium-12 large-12`? I think the common practice is to stop at the smallest relevant size